pyrometheus/chem_expr.py

```diff
@@ -159,15 +159,16 @@ def gen(c, t):
 # {{{ Data-handling helper
 
 def _zeros_like(argument):
-    # FIXME: This mishandles NaNs.
+    # Fixme: This mishandles NaNs.
```

Comment:
The all-caps version is actually intentional and treated as special for example by Vim.
```suggestion
    # FIXME: This mishandles NaNs.
```